tasks/18.txt

```diff
@@ -1,3 +1,31 @@
 Create a SQL query to calcualte total orders count by each day in 1998:
 
-| Order Date | Total Number of Orders |
\ No newline at end of file
+| Order Date | Total Number of Orders |
+______________
+
+db.orders.aggregate([
+    {
+        $project: {
+            "Day": {$dayOfYear: "$OrderDate"},
+            "Year": {$year: "$OrderDate"}
+        }
+    },
+    {
+        $match: {
+            "Year": 1998
+        }
+    },
+    {
+        $group: {
+            _id: "$Day",
+            "Total Number of Orders": {$sum: 1}
+        }
+    },
+    {
+        $project: {
+            _id: 0,
+            "Order Date": "$_id",
+            "Total Number of Orders": 1
+        }
+    },
+])
```

Comment:
there exists a solution with $match before project

Reply:
date can be with time. So just note, the solution with $project, like before is more universals